test/parallel/test-stream-readable-with-unimplemented-_read.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
-require('../common');
+const common = require('../common');
 const stream = require('stream');
 const assert = require('assert');
 
 const readable = new stream.Readable();
 
-assert.throws(() => readable.read(), /ERR_STREAM_READ_NOT_IMPLEMENTED/);
+assert.throws(() => readable.read(), common.expectsError({
```

Comment:
Just as a suggestion - it would be nicer to use
```js
common.expectsError(throwingFn, errorObj)
// instead of
assert.throws(throwingFn, common.expectsError(errorObj))
```